controller/workitem_blackbox_test.go

```diff
@@ -504,6 +506,23 @@ func createOneRandomArea(ctx context.Context, db *gorm.DB) *area.Area {
 	return &ar
 }
 
+func newChildIteration(ctx context.Context, db *gorm.DB, parentIteration *iteration.Iteration) *iteration.Iteration {
+	iterationRepo := iteration.NewIterationRepository(db)
+
+	parentPath := append(parentIteration.Path, parentIteration.ID)
+	itr := iteration.Iteration{
+		Name:    "Sprint 101",
+		SpaceID: parentIteration.SpaceID,
+		Path:    parentPath,
+	}
+	err := iterationRepo.Create(ctx, &itr)
+	if err != nil {
+		fmt.Println("Failed to create iteration.")
```

Comment:
Sure :)

> I did not want these helper methods to actually assert or test.
yeah, i had the same thought initially but later on came to realize that the asserts in the helper methods help nip stuff in the bud before things roll further. :)